sql/src/main/java/io/crate/metadata/Reference.java

```diff
@@ -74,6 +74,7 @@ public String apply(@Nullable Reference input) {
     private RowGranularity granularity;
     private IndexType indexType = IndexType.NOT_ANALYZED;
     private boolean nullable = true;
+    private boolean columnStoreDisabled = false;
```

Comment:
Could we add a comment here for the field, explaining that this means the doc values are disabled?